src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -380,15 +401,15 @@ private JsonObject getAccountForLoan(UUID loanId, String feeType) {
       .getFirst();
   }
 
-  private UUID createLoanAndDeclareItLost() {
+  private IndividualResource createLoanAndDeclareItLost() {
```

Comment:
If the intent is to declare the item lost, I think the individual steps in that process, including checking the item out, can be hidden in this method

```suggestion
  private IndividualResource declareItemLost() {
```